### ISSUE TITLE
Latest provider release (1.3.1) does not include latest master branch changes

### ISSUE THREAD

**Issue by git-ival** [rancher/terraform-provider-rke#349]
Unable to create an `rke_cluster` with the new expected default kubernetes version `v1.23.6-rancher1-1` using the latest release of this provider. This is expected because RKE 1.3.11 defaults to this kubernetes version, see https://github.com/rancher/terraform-provider-rke/pull/340. 

It seems like the latest `master` branch changes were intentionally not included in the 1.3.1 release, is there some cadence that can be assumed for this and future versions to be released?

Example terraform:
```
resource "rke_cluster" "this" {
  ...
  kubernetes_version = "v1.23.6-rancher1-1
  ...
}
```
I receive the following error (abbreviated for the sake of brevity):
`Error: expected kubernetes_version to be one of [v1.22.4-rancher1-1 v1.21.7-rancher1-1 v1.21.6-rancher1-2...], got v1.23.6-rancher1-1`

**Comment by annablender** [comment]
Reference: https://github.com/rancher/terraform-provider-rke/issues/342#issuecomment-1192809924

Hashicorp releases terraform provider RKE v1.3.2, this will be resolved

**Comment by git-ival** [comment]
Fixed with the 1.3.2 release on official Hashicorp Terraform registry
